packages/firestore/src/lite-api/aggregate.ts

```diff
@@ -94,25 +83,24 @@ export function getAggregateFromServerDirect(
 
   return invokeRunAggregationQueryRpc(datastore, aggregateQuery).then(
     result => {
-      const aggregationFields = new Map<string, any>();
-      /**
-       * while getting aggregation fields from server direct, it should have only
-       * one RunAggregationQueryResponse returned.
-       * But we used streaming rpc here, so we will have an array of
-       * (one, or possibly more) RunAggregationQueryResponse. For this specific
-       * function, we get the first RunAggregationQueryResponse only.
-       */
-      for (const [key, value] of Object.entries(result[0])) {
-        aggregationFields.set(key, userDataWriter.convertValue(value));
-      }
+      hardAssert(
+        result[0] !== undefined,
+        'Aggregation fields are missing from result.'
+      );
+
+      const countField = (result[0] as any).count_alias;
```

Comment:
Our linter is complaining about the use of `any` here. What do you think about something like this alternative:

```ts
const counts = Object.entries(result[0])
  .filter(entry => entry[0] == 'count_alias')
  .map(entry => userDataWriter.convertValue(entry[1] as Value));
hardAssert(
  counts.length > 0,
  'count_alias property is missing from result.'
);
const count = counts[0];
```